src/components/common/form/InputRounded.js

```diff
@@ -65,7 +66,12 @@ const getStylesFromProps = ({ theme, disabled }) => {
     fontSize: normalize(14),
     justifyContent: 'center',
     fontWeight: '400',
-    paddingVertical: disabled ? 14 : 10,
+    paddingVertical: getDesignRelativeHeight(
```

Comment:
no need for relative height
why is the padding different for web and android? it should look the same

Reply:
The android padding was because the lineHeight/height of the input is larger in android, changing lineHeight could fix it but it doesn't affect the placeholder, so the padding seems to be the best solution.